services/QuillLMS/spec/models/unit_template_spec.rb

```diff
@@ -323,37 +323,34 @@
 
   describe '#around_save callback' do
     before do
-      $redis.redis.flushdb
-      $redis.multi {
-        $redis.set('beta_unit_templates', 'a')
-        $redis.set('production_unit_templates', 'a')
-        $redis.set('gamma_unit_templates', 'a')
-        $redis.set('alpha_unit_templates', 'a')
-      }
+      Rails.cache.clear
+      Flags::FLAGS.each do |flag|
```

Comment:
I love this update since I bet this used to clear all the flags, but then we added new ones and didn't update this code.